CodeMaid/UI/ToolWindows/Spade/CodeItemTemplates.xaml

```diff
@@ -91,17 +91,51 @@
     </DataTemplate>
 
     <DataTemplate x:Key="NameParametersDataTemplate">
-        <ContentControl x:Name="name"
-                Content="{Binding Converter={x:Static cnv:NameParametersToTextBlockConverter.Default}}"
-                Focusable="False" />
+        <ContentControl x:Name="name" Focusable="False">
+            <ContentControl.Style>

```

Comment:
Does this need to be a Style, with a Setter.. or can the content simply be set directly to a MultiBinding vs. the Binding syntax it was being used before?

e.g.
<ContentControl>
    <MultiBinding>